rome/src/main/java/com/rometools/rome/feed/synd/impl/ConverterForRSS091Userland.java

```diff
@@ -89,6 +76,11 @@ public void copyInto(final WireFeed feed, final SyndFeed syndFeed) {
                 creators.addAll(s);
             }
         }
+
+        final List<Item> items = channel.getItems();
+        if (items != null) {
+            syndFeed.setEntries(createSyndEntries(items, syndFeed.isPreservingWireFeed()));
+        }
```

Comment:
Is this code necessary? Doesn't `ConverterForRSS091Userland` inherit it from `ConverterForRSS090`?